boards/nucleo-l152/include/periph_conf.h

```diff
@@ -34,9 +34,17 @@ extern "C" {
  **/
 #define CLOCK_HSI           (16000000U)             /* frequency of internal oscillator */
 #define CLOCK_CORECLOCK     (32000000U)             /* targeted core clock frequency */
-/* 0: no external low speed crystal available,
- * 1: external crystal available (always 32.768kHz) */
-#define CLOCK_LSE           (1)
+/*
+ * 0: no external low speed crystal available,
+ * 1: external crystal available (always 32.768kHz)
+ *
+ * LSE might not be available by default in early (C-01) Nucleo boards.
+ * If you're sure it is present, define CLOCK_LSE=1 in your project
```

Comment:
If someone with an C-03 board revision follows what is described here, it will result in a failing board (e.g. no stdio for example).
I suggest removing this line until we have a solution with LSE.

Reply:
Hmm... I'm still not completely convinced by this. I think that's obvious that if your board is not working after you set CLOCK_LSE=1 the reason is obviously that LSE is not working properly, so this message still make sense.

Besides, the fact that this board is broken with CLOCK_LSE is a bug, which should be documented somewhere e.g. an issue. Afterwards, of course it needs to be fixed, and it's out of the scope of this PR.

Reply:
ok, but this is not what this comment is saying: 'If you're sure it is present' doesn't mean LSE might be broken.

Reply:
Ok I can add such information.

Reply:
Can you open an issue for this BTW?